src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/time.css.twig

```diff
@@ -0,0 +1,94 @@
+/* Variables */
+
+:root {
+    --color-default: #777;
+    --color-section: #999;
+    --color-event-listener: #00B8F5;
+    --color-template: #66CC00;
+    --color-doctrine: #FF6633;
+    --color-messenger-middleware: #BDB81E;
+    --color-controller-argument-value-resolver: #8c5de6;
+}
+
+/* Legend */
+
+.sf-profiler-timeline .legends .default { border-color: var(--color-default); }
+.sf-profiler-timeline .legends .section { border-color: var(--color-section); }
+.sf-profiler-timeline .legends .event-listener { border-color: var(--color-event-listener); }
+.sf-profiler-timeline .legends .template { border-color: var(--color-template); }
+.sf-profiler-timeline .legends .doctrine { border-color: var(--color-doctrine); }
+.sf-profiler-timeline .legends .messenger-middleware { border-color: var(--color-messenger-middleware); }
+.sf-profiler-timeline .legends .controller-argument-value-resolver { border-color: var(--color-controller-argument-value-resolver); }
+
+svg {
```

Comment:
I don't think you should apply styles on all `svg` elements, for things meant for the timeline SVG. Icons in the UI are also using SVG, so this could create issues in the future.

Reply:
Fixed, thanks.